03_Basic SQL/Scripts/1.1.1_Orders_after_6.5.98.sql

```diff
@@ -0,0 +1,3 @@
+﻿SELECT OrderID, ShippedDate, ShipVia 
+  FROM [dbo].[Orders]
+  WHERE ShippedDate>=convert(datetime,'05.06.1998', 121) AND ShipVia>=2
```

Comment:
I think with SQL Server there is no need to convert string to datetime, as you can just use dates in queries in this format: `'1998-12-31'`. They will also work with `>`, `<`, `=`

Reply:
fixed